nursery/resolve-function-by-fnv-hash.yml

```diff
@@ -0,0 +1,21 @@
+rule:
+  meta:
+    name: resolve function by FNV hash
```

Comment:
can you please add a `description:` that this is `FNV-1a`
alternatively modify the name (and rename the file)

Reply:
Done. Hope that's enough!

Reply:
almost, the file needs to be renamed to `resolve-function-by-fnv-1a-hash.yml` and I think reformatted, you can test it locally using `scripts/lint.py`

Reply:
Oh, didn't notice there was an enforced linting rule. Re-saved as LF-only and whatnot. Should pass the linting now.

Reply:
...wait, forgot to add the renamed file.